storage_module.rb

```diff
@@ -0,0 +1,17 @@
+require 'JSON'
```

Comment:
- [ ] Here happens the same:

![image](https://github.com/mahammad-mostafa/school-library/assets/77069011/00cfd2e3-b478-400f-83a7-19159d2e955c)

Please consider changing it to lowercase! 🤓 🔧